src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -1191,6 +1173,68 @@ private static boolean shallUpdate(final World world, final int averageTicks)
         return world.getWorldTime() % (world.rand.nextInt(averageTicks * 2) + 1) == 0;
     }
 
+    private boolean areAllColonyChunksLoaded(@NotNull final TickEvent.WorldTickEvent event)
+    {
+        final int distanceFromCenter = Configurations.gameplay.workingRangeTownHall + 48 /* 3 chunks */ + 15 /* round up a chunk */;
+        for (int x = -distanceFromCenter; x <= distanceFromCenter; x += 16)
+        {
+            for (int z = -distanceFromCenter; z <= distanceFromCenter; z += 16)
+            {
+                if (!event.world.isBlockLoaded(new BlockPos(getCenter().getX() + x, 128, getCenter().getZ() + z)))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 128 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS109)